CONTRIBUTING.adoc

```diff
@@ -30,14 +27,12 @@ https://github.com/openshift/origin/releases[the Releases page].  These are
 Linux, Windows, or Mac OS X 64bit binaries (note that Mac and Windows are
 client only). You'll need Docker installed on your local system
 (see https://docs.docker.com/installation/[the installation page] if you've
-never installed Docker before).
+never installed Docker before and you're not on RHEL/CentOS/Fedora).
 
 The tar file for each platform contains a single binary `openshift` which is
 the all-in-one OpenShift installation.
 
-* Use `sudo openshift start` to launch the server.  Root access is required to
-  create services due to the need to modify IPTables.
-  See issue: https://github.com/kubernetes/kubernetes/issues/1859.
+* Use `oc cluster up` to launch the server.
```

Comment:
> What are the ways of building and rebuilding images? 

The ones you have mentioned are the ways to build all of them.

You can re-build them with `hack/build-local-images.py`.